docs/BUILD.bazel

```diff
@@ -1,14 +1,21 @@
 # This load statement must be in the docs/ package rather than anything users depend on
 # so that the dependency on stardoc doesn't leak to them.
-load("@aspect_bazel_lib//lib:docs.bzl", "stardoc_with_diff_test", "update_docs")
+load("@aspect_bazel_lib//lib:docs.bzl", "update_docs")
+load(":stardoc_with_diff_test.bzl", "stardoc_with_diff_test")
```

Comment:
how about we just always put `-//docs/...` in the target pattern for Bazel 6 in the workflow, then you don't need to use tags and don't have to introduce a fork of our rule?

I think it's never the case that we want multi-version testing of our docgen

Reply:
That would complicate the GHA code. The tag is simpler.

Reply:
I put up https://github.com/aspect-build/bazel-lib/pull/746 since the tags not merging is a legit bug